README.md

```diff
@@ -1,3 +1,49 @@
 # FLorist
 
-TODO
+A platform to launch federated learning (FL) training jobs.
+
+## Setting up
+
+### Install dependencies
+
+To install the project dependencies, first you need to create a virtual environment.
+The easiest way is by using the [virtualenv](https://pypi.org/project/virtualenv/) package:
+
+```shell
+virtualenv venv
+source venv/bin/activate
+```
+
+We use [Poetry](https://python-poetry.org/) to manage back-end dependencies:
+
+```shell
+pip install --upgrade pip poetry
+poetry install
```

Comment:
I got a strange looking warning when running this, not sure if you ran into the same thing and just wanted to flag it. It seems to be in regards to poetry expecting a certain format to the directory. This [issue](https://github.com/python-poetry/poetry/issues/8662) seems to describe it. I guess it comes down to using poetry for dependency management vs poetry for dependency management and packaging. I guess we are only going to be using for dependency management  so this doesn't seem relevant.

Reply:
Thanks for catching that. I just ignored the warning but reading that issue, poetry expects to have a folder with the package name so I've changed #3 to reflect that and the warning doesn't appear anymore.